merklehash/src/lib.rs

```diff
@@ -89,25 +86,25 @@ impl Tree {
 
                 // Insert the input into the merkle tree.
                 for i in 0..nodes_size_of_lowest_row {
-                    nodes[first_index_in_lowest_row + i] = input[i]
+                    nodes[first_index_in_lowest_row + i] = input[i].clone()
```

Comment:
Before there is no clone, does the H256 have copy trait?

Reply:
Yes, and clippy will not allow `clone()` here.